rmap-loader-deposit-disco/src/main/java/info/rmapproject/loader/deposit/disco/DiscoDepositConsumer.java

```diff
@@ -139,8 +138,7 @@ public void accept(HarvestRecord record) {
             if (response.getStatusLine().getStatusCode() == 201) {
                 harvestRegistry.register(
                         record.getRecordInfo(),
-                        URI.create(response.getFirstHeader("Location")
```

Comment:
Funnily enough, this is what @emetsger has been working on for the last few weeks - you can spin up a working test environment, it stores the test data until you close it down when you're done.  https://github.com/rmap-project/rmap/blob/master/DEVELOPER.md